supabase/migrations/20260814230000_f010b_rls_lockdown.sql

```
@@ -0,0 +1,150 @@
+alter table public.profiles enable row level security;
+
+drop policy if exists "Anyone can view profiles" on public.profiles;
+drop policy if exists "Enable read access for all users" on public.profiles;
+drop policy if exists "profiles_select_manager_or_own" on public.profiles;
+create policy "profiles_select_manager_or_own"
+    on public.profiles
+    for select
+    to authenticated
+    using (
+        id = auth.uid()
+        or exists (select 1 from public.profiles p where p.id = auth.uid() and p.role = 'MANAGER')
```

SQLFluff (4.2.2)

[error] The 'where' keyword should always start a new line.

(LT14)